### ISSUE TITLE
abstract method shows wrong parameter info

### ISSUE THREAD

**Issue by myphysicslab** [jleyba/js-dossier#30]
Abstract methods show up like this in docs:

```
addPath(arg0)

Specified by: myphysicslab.lab.engine2D.Edge

Parameters
arg0...?
```

instead of showing the correct parameter information from the Edge interface:

```
addPath(context)

Add this Edge to the currently open path of the JavaScript canvas context. The Edge should be rendered in local body coordinates.

Parameters
contextCanvasRenderingContext2D
the JavaScript canvas context to draw this object into

```

Here is some of the code involved:

```javascript
/** 
* @interface
*/
myphysicslab.lab.engine2D.Edge = function() {};

/** Add this {Edge} to the currently open path of the JavaScript canvas context.
The {Edge} should be rendered in local body coordinates.
@method
@param {!CanvasRenderingContext2D} context the JavaScript canvas context to draw this
object into
*/
Edge.prototype.addPath

/**
* @param {!myphysicslab.lab.engine2D.Polygon} body
* @param {!myphysicslab.lab.engine2D.Vertex} vertex1 the previous vertex, in body
  coords; matches the next (second) vertex of the previous edge
* @param {!myphysicslab.lab.engine2D.Vertex} vertex2 the next vertex, in body coords
* @constructor
* @struct
* @implements {myphysicslab.lab.engine2D.Edge}
*/
myphysicslab.lab.engine2D.AbstractEdge = function(body, vertex1, vertex2) {
  // code omitted...
};

/** @inheritDoc */
AbstractEdge.prototype.addPath = goog.abstractMethod;
```

**Comment by jleyba** [comment]
I've run into some ambiguity over when to use the documentation from a super type.  Suppose we have:

```
/** @constructor */
var Greeter = function() {};

/**
 * Generates a greeting.
 * @param {string} name Name of the person to greet.
 * @return {string} The greeting.
 */
Greeter.prototype.greet = goog.abstractMethod;

/**
 * @constructor
 * @extends {Greeter}
 */
var CustomGreeter = function() {};

/**
 * Generates a custom greeting.
 * @override
 */
CustomGreeter.prototype.greet = function() {};
```

If `CustomGreeter.prototype.greet` simply had `/** @override */`, we can easily grab the docs from the super type. But above, there is a block description. Do we

1. Use the block comment and inherit the documentation for the annotations. The docs will also include an "Overrides: Greeter#greet" link.
2. Use the block comment, but leave everything else empty with an "Overrides: Greeter#greet" link.

What if the comment overrides the documentation just for the `@param`:
```
/** 
 * @param {string} name The name of a funny person.
 * @override
 */
CustomGreeter.prototype.greet = function() {};
```

Here, do we use the `@param` and inherit everything else, or just include the "Overrides..." link?

**Comment by myphysicslab** [comment]
I've been assuming that I cannot modify the documentation when I'm using @inheritDoc (or @override which I guess is the same thing).  So this wouldn't be much of an issue for me.

I've noticed that I can provide a duplicate of the type information, which can have different documentation, but I assume it should have the same signature (same parameter and return types).  That's a closure-compiler question which I don't know the answer to.  Seems like the compiler should object if the types are different.

If there is a way to provide a warning when you notice this happening, that would be good enough for now I would think.

**Comment by jleyba** [comment]
This is included in the v0.7.0 release

**Comment by myphysicslab** [comment]
Confirmed this case is fixed, however see issue #35 for a similar case that is still wrong.
